src/components/PetsBreadCrumbs.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import {
+  Breadcrumbs,
+  BreadCrumbLinkItem
+} from "@baltimorecounty/dotgov-components";
+
+const PetBreadCrumbs = props => {
+  const { animalName } = props;
+
+  return (
+    <Breadcrumbs>
+      <BreadCrumbLinkItem text="Departments" link="/departments/health" />
+      <BreadCrumbLinkItem text="Animal Services" link="/departments/health" />
+      <BreadCrumbLinkItem
```

Comment:
this link would need to point to the adoptable pets list. So you would probably need to use react router Link here.